packages/devextreme/js/ui/map.d.ts

```diff
@@ -205,6 +205,24 @@ export interface dxMapOptions extends WidgetOptions<dxMap> {
      * @public
      */
     focusStateEnabled?: boolean;
+    /**
+     * @docid
+     * @default { mapId: '', useAdvancedMarkers: true }
```

Comment:
We declare defaults for mapId, and UseAdvancedMarkers inside. Should we declare default object with all properties? Moreover, mapId and useAdvancedMarkes are optional here

Reply:
>Should we declare default object with all properties?
We do this in similar cases
>Moreover, mapId and useAdvancedMarkes are optional here
That doesn't mean they don't have a default values, which we want to be shown in doc

Reply:
I mean that each property already has its own default, but we are duplicating and listing these defaults at the object level. This could lead to maintenance issues, and if there were more than two nested properties, it would look odd. But okay, I see that this is already being done in the code, so let's leave it as is.